src/types/types.js

```diff
@@ -878,13 +878,13 @@ export type EdgeSwapApproveOptions = {
  * If a provider can satisfy a request, what is their price?
  */
 export type EdgeSwapQuote = {
+  +swapInfo: EdgeSwapInfo,
```

Comment:
Actually, wait! The core is responsible for making sure this exists, even if the plugins are outdated. This means we need to upgrade the quote on the way out the door, in the same way we add the request itself if that is missing.